source/Ship.cpp

```diff
@@ -196,20 +196,21 @@ namespace {
 
 
 // Construct and Load() at the same time.
-Ship::Ship(const DataNode &node)
+Ship::Ship(const DataNode &node, const bool isSavedShip)
```

Comment:
~~surely making a bool const here is pushing it a bit?~~
also you didnt make it const in the .h, but I like it that way better

Reply:
Variables that are passed by value never get marked as const in headers.

And I think I mentioned I liked making things const if I could already. ;)

Reply:
we never have const variables that arent classes tho as far as I'm aware

Reply:
They show up here and there.

```
const StellarObject *AI::FindLandingLocation(const Ship &ship, const bool refuel)
bool AI::CanHelp(const Ship &ship, const Ship &helper, const bool needsFuel) const
bool AI::HasHelper(const Ship &ship, const bool needsFuel)
```

Reply:
alr I surrender
~~just dont make const ref to bools~~